e2e/scripts/actions/env_path.test

```diff
@@ -0,0 +1,30 @@
+#!/singularity/test
+# env path tests
+
+source common.func
+
+default_image="docker://alpine:3.8"
+default_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+custom_image="docker://godlovedc/lolcow"
+custom_path="/usr/games:${default_path}"
+partial_path="/foo"
+overwritten_path="/usr/bin:/bin"
+
+stdout-contains "DefaultPath"           "${default_path}"                   singularity exec ${default_image} printenv PATH
```

Comment:
How do you test `stdout` and `stderr` at the same time? This is something that is not uncommon to do.

Reply:
Will enhance this test builtin to allow use of "combined" stream argument